yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```

Comment:
empty?

Reply:
Yes. IDK why there are 2 `package.json files`: one for the whole repo (https://github.com/aion-dk/election-verification-site/blob/main/package-lock.json) and one for the "client" dir (where the actual app is). The one outside doesn't have anything installed, so the `yarn.lock` file is also empty.